extensions/positron-python/python_files/positron/positron_ipykernel/positron_ipkernel.py

```diff
@@ -438,6 +438,27 @@ def __init__(self, **kwargs) -> None:
             module="jedi",
         )
 
+        # TODO: We could move this to its own function in a separate module.
+        # Patch holoviews to use our custom notebook extension.
+        try:
+            import holoviews
```

Comment:
I think this error means that we need to add holoviews to the `pinned-test-requirements.txt` file.